### PR TITLE
RPE-1061 Add Java 11 image

### PR DESCRIPTION
**Pull request by timja**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPE-1061

### Change description ###
Add java 11 base image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: 11/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+COPY group /etc/group
+COPY passwd /etc/passwd
+
+COPY --chown=hmcts:hmcts app/ /opt
+
+WORKDIR /opt/app
+USER $APP_USER
+
+COPY applicationinsights-agent-2.3.1.jar /opt/app
```
Comment (timja) [inline 11/Dockerfile]: Not sure if this is the best approach, but there's no wget or curl installed, and not sure if we would want to add it for this

Also app insights agent is broken on java 11, looking at fixing it.
see https://github.com/Microsoft/ApplicationInsights-Java/issues/897

File: 11/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+#   -XX:+UseParallelOldGC 
+# - Grow the heap when only 20% is free and shrink it when more than 40% is free:
+#   -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 
+# - A hint to the virtual machine that up to 20% of time can be spent in GC (this will save memory at the cost of some time):
+#   -XX:GCTimeRatio=4 
+# - Timing goal check should be mostly based on to the current GC execution time (90% weight):
+#   -XX:AdaptiveSizePolicyWeight=90
+# - Use a small min heap size (try to save memory):
+#   -Xms128M
+#ONBUILD ENV JAVA_TOOL_OPTIONS "-XX:InitialRAMPercentage=20.0 -XX:MaxRAMPercentage=65.0 -XX:MinRAMPercentage=10.0 -XX:+UseParallelOldGC -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -Xms128M ${JAVA_OPTS} ${JAVA_AGENT_OPTIONS}"
```
Comment (timja) [inline 11/Dockerfile]: Temporarily disabled because the agent is broken

File: 11/Dockerfile (hunk 1 above)
Comment (timwebster9) [reply 11/Dockerfile]: would a multi-stage build help here? (i.e. use something with cURL in it to download the binary)

Comment (timja) [reply 11/Dockerfile]: hmm yeah but i think that would need to be in the consumer image not this one

**Pull request merged**